docs/source_values.rst

```diff
@@ -65,13 +70,16 @@ device will iterate over the values of the device provided, setting the
 device's value to each element at a rate specified in the
 :attr:`~SourceMixin.source_delay` property.
 
-.. image:: images/source_values.*
+.. image:: images/source_values/source_values.*
     :align: center
 
 The most common use case for this is to set the source of an output device to
-the values of an input device, like the example above. A more interesting
+match the values of an input device, like the example above. A more interesting
 example would be a potentiometer controlling the brightness of an LED:
 
+.. image:: images/source_values/pwmled_pot.*
+    :align: center
+
 .. literalinclude:: examples/pwmled_pot.py
 
 The way this works is that the device's :attr:`~ValuesMixin.values` property is
```

Comment:
"The way this works is that the device's `values` property is used to feed values into the device."
sounds a little bit confusing (two mentions of "device"). Perhaps
"The way this works is that the pot device's `values` property is used to feed values into the led device's `source` property." ?

Reply:
I've changed "device" to "input device" and "output device". Much better.